<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1400</width>
    <height>900</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1400</width>
    <height>900</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1400</width>
    <height>900</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#QPushButton {
	
	background-color: rgb(0, 115, 97);
	color: rgb(129, 61, 156);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>120</horstretch>
     <verstretch>80</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>1400</width>
     <height>900</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1400</width>
     <height>900</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">#centralwidget {
	background-color: rgb(0, 115, 97);
	background: url(:/background.jpg);
}

QPushButton { 
	border-radius: 5px;	
	background-color: rgb(12, 71, 24);
	border-style:outset;
	border-color: rgb(0, 90, 75);
	color: rgb(255, 255, 255);	
	font: 14pt ;
}

QPushButton:hover { 
	
	color: rgb(0, 0, 0);
	background-color: rgb(150, 224, 166);
	border-style: outset;
	border-color: rgb(0, 90, 75);
	border-width: 1px;
}

QScrollBar {
	border:none;
	background-color: rgb(13, 80, 10);
}

QScrollBar::handlebar:vertical {
	
	
	background-color: rgb(13, 44, 19);
	border: none;

}

QScrollBar::handlebar:vertical:pressed {
	
	background-color: rgb(0, 129, 28);
	border: none;

}


</string>
   </property>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>20</y>
      <width>240</width>
      <height>300</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>240</width>
      <height>300</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>240</width>
      <height>300</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">#groupBox {

border-radius: 10px;

	background-color: rgba(45, 187, 50, 100);
}

</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>191</width>
       <height>262</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="bottomMargin">
           <number>20</number>
          </property>
          <item>
           <widget class="QPushButton" name="LoadButton">
            <property name="styleSheet">
             <string notr="true">font: &quot;:/fonts/Font.ttf&quot;</string>
            </property>
            <property name="text">
             <string>LOAD MAP</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="SaveButton">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>SAVE MAP</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="topMargin">
           <number>20</number>
          </property>
          <item>
           <widget class="QPushButton" name="StartStopButton">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>START</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="RestartButton">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>RESTART</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="ClearButton">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>CLEAR</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="speed_label">
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>SPEED:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(9, 71, 23);
color: rgb(255, 255, 255);
selection-background-color: transparent;</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="CustomView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>1100</width>
      <height>850</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>110</horstretch>
      <verstretch>85</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>1100</width>
      <height>850</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1100</width>
      <height>850</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">
border-radius: 10px;

background-color: rgba(50, 144, 48, 163);

</string>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomView</class>
   <extends>QGraphicsView</extends>
   <header>customview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
